bin/pycheck

```diff
@@ -17,25 +17,18 @@ cd "$(dirname "$0")/.."
 
 . misc/shlib/shlib.bash
 
-mypy_folders=(ci misc/python)
+PYRIGHT_VERSION="1.1.323"
+typecheck_cmd="npx pyright@$PYRIGHT_VERSION"
+
+python_folders=(ci misc/python)
 flake8_folders=(ci misc/python test)
 
-try bin/pyactivate -m mypy "${mypy_folders[@]}"
-if ! try_last_failed; then
-    # mzcompose.py files need to be passed to mypy individually to avoid
-    # duplicate module warnings. We don't check these if the previous `mypy`
-    # invocation failed, though, or we'll get a ton of duplicate warnings.
-    for file in $(git_files "**/mzcompose.py"); do
-        try bin/pyactivate -m mypy "$file"
-    done
-
-    for file in $(git_files "test/**.py"); do
-        if [[ $file == *"mzcompose"* ]]; then
-            continue
-        fi
-        try bin/pyactivate -m mypy "$file"
-    done
-fi
+compose_files=$(git_files "**/mzcompose.py")
+test_files=$(git_files "test/**.py")
```

Comment:
Can pyright not handle being presented `test` as a directory? Bummer.

Reply:
Honestly I didn't try it -- let me give it a shot. I've been trying to preserve as much of the previous semantics as possible 😉